### ISSUE TITLE
enhancement in the submit the answer 

### ISSUE THREAD

**Issue by enochCodes** [Lafetz/multiplayer-trivia#1]
in the select your answer section example if i set timer 10 min the answer will submitted automatically with that time interval but it must be needed force submit there because it i set the timer 20 or 30 sec i must to wait that long so it needed enhancement there force submit.

**Comment by Lafetz** [comment]
you're right, i didn't consider this since the game was designed as multiplayer with the timer controlling gameplay. it's been a while since i last touched the code, but i'll revisit it and add this feature.

**Comment by Lafetz** [comment]
> in the select your answer section example if i set timer 10 min the answer will submitted automatically with that time interval but it must be needed force submit there because it i set the timer 20 or 30 sec i must to wait that long so it needed enhancement there force submit.

if you'd like to fix it or contribute, feel free to jump in!
